lib/server.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+require('dotenv').config();;
+const express = require('express');
+const app = express()
```

Comment:
This is why router doesn't work, if you switch this to router it should work

Reply:
Roger that! Fixed it overall! Still not seeing my `logger` middleware or other middleware in console...hmm. That works before when using app.use. I shall investigate! 👍